src/combination.jl

```diff
@@ -15,11 +16,13 @@ function BlissCombination(p1::Array{Float64, 2}, p2::Array{Float64, 2}, n::Int)
     param2[3, :] .= p2[3, :]
     param2[4, :] .= p2[4, :]
```

Comment:
Aren't these parameters already only for specific concentrations? Does this function work if you set `n` to values other than 8?

Reply:
I had the option before that could plot parameters for any number of points between the min and max of the concentration. But I don't know if we will need that option later or not.